Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageCollectionViewCell.swift

```diff
@@ -74,3 +74,54 @@ public final class _ChatMessageCollectionViewCell<ExtraData: ExtraDataTypes>: _C
         return preferredAttributes
     }
 }
+
+/// The cell that displays the message content of a dynamic type and layout.
+/// Once the cell is set up it is expected to be dequeued for messages with
+/// the same content and layout the cell has already been configured with.
+public final class _ChatMessageTableViewCell<ExtraData: ExtraDataTypes>: _TableViewCell {
```

Comment:
Maybe remove the collection view cell code and rename the file? or create separated file for the table view cell.

Reply:
Yes, this was the plan 🙂 Done ✅